src/dmtcp_restart.cpp

```diff
@@ -725,9 +725,9 @@ int main(int argc, char** argv)
       printf("\nProcess uid (%d) doesn't match uid (%d) of\n" \
              "checkpoint image (%s).\n" \
 	     "This is dangerous.  Aborting for security reasons.\n" \
-             "If you still want to do this (at your own risk),\n" \
-             "  then modify dmtcp/src/%s:%d and re-compile.\n",
-             getuid(), buf.st_uid, restorename.c_str(), __FILE__, __LINE__ - 7);
+             "If you still want to do this, then re-run dmtcp_restart\n" \
+             "  with the --run-as-root flag.\n",
```

Comment:
I've now replaced --run-as-root with --no-strict-checking in the second commit.  (This line note refers to the first commit.)